library/common/extensions/filters/http/platform_extension/filter.proto

```diff
@@ -0,0 +1,9 @@
+syntax = "proto3";
+
+package envoymobile.extensions.filters.http.platform_extension;
+
+import "validate/validate.proto";
+
+message Bridging {
+  string name = 1 [(validate.rules).string.min_bytes = 1];
+}
```

Comment:
Can we document how this works/is intended to be used?

Reply:
This particular part won't ever really be publicly exposed/used. But I do intend to make a documentation PR that fully describes how to create and use platform extensions.